src/menu/menu.js

```diff
@@ -1,81 +1,150 @@
 import { fetchData } from "../helpers.js";
 
-export function getMenuTemplate() {
-  return ` <h1>Menu</h1>`;
+export function getRestaurantsDetailsTemplate() {
+  return `
+  <div id="loader" class="center"></div>
+  <h1>Restaurant Details</h1>
+  <div class="hide-content">
+    <h4 id="name"></h4>
+    <h4 id="address1-label" style="display: inline-block;">Address I: </h4>
+    <p id="address1" style="display: inline-block;"></p>
+    <br>
+    <h4 id="address2-label" style="display: inline-block;">Address II: </h4>
+    <p id="address2" style="display: inline-block;"></p>
+    <br>
+    <h4 id="latitude-label" style="display: inline-block;">Latitude:</h4>
+    <p id="latitude" style="display: inline-block;"></p>
+    <br>
+    <h4 id="longitude-label" style="display: inline-block;">Longitude:</h4>
+    <p id="longitude" style="display: inline-block;"></p>
+    <h1>Menu</h1>
+  </div>
+ `;
 }
 
-export function getRestaurantMenu(id) {
-  fetchData(
-    `https://private-anon-7231255228-pizzaapp.apiary-mock.com/restaurants/${id}/menu?category=Pizza&orderBy=rank`
-  ).then((result) => {
-    const menuBody = document.getElementById("main");
-
-    let totalPrice = 0;
-    let totalQuantity = 0;
-
-    result.forEach((item) => {
-      const row = document.createElement("tr");
-      const name = document.createElement("td");
-      const topping = document.createElement("td");
-      const price = document.createElement("td");
-      const addToCartBtn = document.createElement("button");
-
-      name.innerText = item.name;
+export function getMenuTemplate() {
+  return `
+  <table>
+  <tbody id="main"></tbody>
+  <tfoot>
+    <tr id="total-row-quantity">
+      <td colspan="2" id="total-quantity">Total quantity: 0</td>
+    </tr>
+    <br>
+    <tr id="total-row-price">
+      <td colspan="2" id="total-price">Total price: $0.00</td>
+    </tr>
+  </tfoot>
+  </table>
+  `;
+}
 
-      topping.innerText = item.topping?.join(", ") || " ";
-      price.innerText = ` $ ${item.price}`;
-      addToCartBtn.innerText = "Add to cart";
+export function getRestaurantDetailsByID(id) {
+  const loader = document.getElementById("loader");
+  const main = document.getElementById("main");
+  const mainDiv = document.querySelector(".hide-content");
+
+  let restaurantData;
+  let menuData;
+
+  loader.classList.remove("hidden");
+  main.classList.add("hidden");
```

Comment:
You can just add `hidden` class in the template because this is the initializer function, it's being called once.
All these classes being added/removed before the API calls can just be done in the templates as you like.